packages/design-system/src/components/tooltip.tsx

```diff
@@ -145,7 +148,8 @@ export const InputErrorsTooltip = ({
       {...rest}
       content={content ?? " "}
       open={errors !== undefined && errors.length !== 0}
-      side="right"
+      side={side ?? "right"}
```

Comment:
Probably because the tooltip component is using `top` as default here
https://github.com/webstudio-is/webstudio/blob/main/packages/design-system/src/components/tooltip.tsx#L103-L111
```jsx
<TooltipPrimitive.Portal>
          <Content
            ref={ref}
            side="top"
            align="center"
            sideOffset={2}
            collisionPadding={8}
            arrowPadding={8}
            {...props}
          >
```

Reply:
right, but then why changing the default we use system-wide instead of passing what you need in your case?

Reply:
Since the tooltips are side by side, it's kinda blocking what's on the right side. If someone is editing the content after adding initially. It's blocking to see what's on the right input.

Reply:
right, because its specifically the case with forms